src/aelfrice/hook.py

```
@@ -836,6 +836,30 @@ def user_prompt_submit(
+            payload_obj: Any = json.loads(raw) if raw.strip() else {}
+            if isinstance(payload_obj, dict):
+                payload_dict = cast(dict[str, object], payload_obj)
+                ck_body = _maybe_run_ups_cadence_checkpoint(
+                    payload_dict, session_id or "", serr,
+                )
+                if ck_body:
+                    cadence_checkpoint_block = (
+                        f"<cadence-checkpoint>\n{ck_body}\n</cadence-checkpoint>"
+                    )
+        except Exception:
+            pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)